### ISSUE TITLE
[Enhancement] Enable use of ngx-ui-scroll on web components

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#1874]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
It seems like the `ngx-ui-scroll` library we have decided to has some issues with rendering items that are web-components, as they have their own lifecycle that affects how/when they render. We should create some functionality that extends the virtual scroll library ensuring that they are actually rendered before being 'consumed' by ngx-ui-scroll. 

The issue is replicated here, and we should test a solution in this simple context before moving on to integrating with kirby components. 

1. Go to https://stackblitz.com/edit/angular-ivy-ppnmpb?file=src%2Fapp%2Fapp.component.html
2. See how a list with ion-item (web component) only fetches the buffer, but no more items when scrolling
3. See how a standard list fetches new items on scroll

## Describe the solution you'd like
We can probably register and await the `onComponentReady()` lifecycle method similar to how we handle web components in our TestHelper. Then we can use the `Adapter.check()` functionality in ngx-ui-scroll to fetch new items. 

## Have you considered any alternatives?
We could investigate the use of other libraries if this does not work, or alternatively consider if we can live with a timeout before fetching elements, which might alleviate the issue. 

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
This is on hold. Turns out https://material.angular.io/cdk/scrolling/overview works out of the box with our components. Unfortunately, only fixed item size is supported, but custom size strategies can be implemented, and have been by the community as seen in: https://github.com/angular/components/issues/10113#issuecomment-778660378
